content/blog/TinaCMS-Version-21.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: TinaCMS Version 2.1
+date: '2024-07-24T14:00:00.000Z'
+last_edited: '2024-07-24T14:00:00.000Z'
+author: Jack Pettit
+---
+
+We are excited to announce the release of `tinacms@2.1.0` and `@tinacms/cli@1.5.51`!
+
+### What's New:
+
+1. **Removed Deprecated `useTina` Hook**
```

Comment:
Is there another name we can use for the deprecated `useTina` hook or a link to the specific one that is being removed? Might be a bit confusing to some people.

Reply:
Yea I wasn't sure how we should have this. Maybe I'll just say we have removed some deprecated code

Reply:
✅